ci_framework/roles/edpm_build_images/tasks/add_cert.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Install libguestfs packages
+  ansible.builtin.package:
+    name:
+      - libguestfs
+      - libguestfs-tools
+      - libguestfs-tools-c
+    state: present
+
+- name: Install cert rpm
+  ansible.builtin.shell:
+    cmd: >-
+      virt-customize -a "{{ cifmw_edpm_build_images_basedir }}/{{ cifmw_discovered_image_name }}"
+      --run-command '{{ cifmw_edpm_build_images_cert_install_command }}'
+  environment:
+    LIBGUESTFS_BACKEND_SETTINGS: force_tcg
```

Comment:
afair.. force_tcg makes use of qemu instead of kvm and impact performance, Do we really need this?

Reply:
we removed force_tcg in the latest patchset.